### DISCUSSION TITLE
Plugin to generate yaml file directly from Azure

### DISCUSSION THREAD

**Discussion by cvallesi-kainos** [Green-Software-Foundation/discussions#169] [Ask Anything]
Good morning everyone, we are trying to integrate the IF in our development lifecycle and with large or complex infrastructures it become really complicated to write the yaml files to submit to the IF script.

So, we are trying to create something that retrieve the necessary information directly from a project and generate the yaml file automatically, or at least for whatever information can be retrieved automatically. We are targeting Azure and just VMs for the moment but the idea, if possible, is to extend it to the relevant other products in Azure (functions, etc) and then expand to AWS and GCP as well.

We are aware of an [abandoned plugin](https://github.com/Green-Software-Foundation/if-unofficial-plugins/tree/main/src/lib/azure-importer) and we are going to test it this week but it would be great to have some input from others that have tried this already if there are any issues we should be aware of.

**Comment by jmcook1186** [comment]
Hi @cvallesi-kainos sorry to have missed this message - we have an updated azure importer built by a team at microsoft - you can find it here: https://explorer.if.greensoftware.foundation/
